### PR TITLE
Make deployer less noisy about successful uploads by default

### LINKED ISSUE #1500: Make deployer less noisy about successful uploads by default
We log every single redirect and every single file upload to stdout in the GitHub Action workflow. 
The problem with this is that it's so incredibly noisy that you can't see any problems/errors. And it's very hard to scroll to see the interesting outputs like the summary. 

For example, I dare you to find out what the actual error was on https://github.com/mdn/yari/runs/1281942720?check_suite_focus=true

### PR DESCRIPTION
**Pull request by peterbe**
Fixes #1500

- [x] Expose it as possible input for the Production Build workflow

### CODE REVIEW

File: .github/workflows/production-build.yml [hunk 1]
```diff
@@ -97,6 +102,7 @@ jobs:
           echo "Trigger notes: ${{ github.event.inputs.notes }}"
           echo "Trigger archived-content: ${{ github.event.inputs.archived_content }}"
           echo "Trigger translated-content: ${{ github.event.inputs.translated_content }}"
+          echo "Trigger log_each_successul_upload: ${{ github.event.inputs.log_each_successul_upload }}"
```
Comment (escattone) [inline .github/workflows/production-build.yml:105]:
```suggestion
          echo "Trigger log_each_successful_upload: ${{ github.event.inputs.log_each_successful_upload }}"
```

File: .github/workflows/production-build.yml [hunk 2]
```diff
@@ -174,6 +180,7 @@ jobs:
           cd deployer
           export DEPLOYER_BUCKET_NAME=mdn-content-dev
           export DEPLOYER_BUCKET_PREFIX=${{ github.event.inputs.deployment_prefix }}
+          export DEPLOYER_LOG_EACH_SUCCESSUL_UPLOAD=${{ github.event.inputs.log_each_successul_upload }}
```
Comment (escattone) [inline .github/workflows/production-build.yml:183]:
```suggestion
          export DEPLOYER_LOG_EACH_SUCCESSFUL_UPLOAD=${{ github.event.inputs.log_each_successful_upload }}
```

File: deployer/src/deployer/constants.py [hunk 3]
```diff
@@ -31,3 +31,8 @@
     default=not sys.stdout.isatty() or bool(json.loads(os.environ.get("CI", "false"))),
     cast=bool,
 )
+
+# If true, it will log every successul upload task as it happens.
+LOG_EACH_SUCCESSUL_UPLOAD = config(
+    "DEPLOYER_LOG_EACH_SUCCESSUL_UPLOAD", default=False, cast=bool
+)
```
Comment (escattone) [inline deployer/src/deployer/constants.py:38]:
```suggestion
LOG_EACH_SUCCESSFUL_UPLOAD = config(
    "DEPLOYER_LOG_EACH_SUCCESSFUL_UPLOAD", default=False, cast=bool
)
```

File: deployer/src/deployer/upload.py [hunk 4]
```diff
@@ -15,6 +15,7 @@
     DEFAULT_CACHE_CONTROL,
     HASHED_CACHE_CONTROL,
     MAX_WORKERS_PARALLEL_UPLOADS,
+    LOG_EACH_SUCCESSUL_UPLOAD,
```
Comment (escattone) [inline deployer/src/deployer/upload.py:18]:
```suggestion
    LOG_EACH_SUCCESSFUL_UPLOAD,
```

File: deployer/src/deployer/upload.py [hunk 5]
```diff
@@ -35,7 +36,7 @@ def log_task(task):
         else:
             # With file upload tasks, the error message says it all.
             log.error(task.error)
-    else:
+    elif LOG_EACH_SUCCESSUL_UPLOAD:
```
Comment (escattone) [inline deployer/src/deployer/upload.py:39]:
```suggestion
    elif LOG_EACH_SUCCESSFUL_UPLOAD:
```

**Review by escattone — CHANGES_REQUESTED** [review]
@peterbe Just a typo (missing "f" in "successul").

**Review by escattone — APPROVED** [review]
Thanks @peterbe.

**Comment by peterbe** [comment]
It worked!
Screen Shot 2020-10-21 at 8 05 23 AM

**Pull request merged**